### DISCUSSION TITLE
How to Build Quasar SPA into Laravel Folder

### DISCUSSION THREAD

**Discussion by mecjos** [quasarframework/quasar#12738] [General - Components / Directives / etc] [answered]
Hi,.. I work on frontend and backend in seperate folders and I want to build my quasar spa directly into laravel's public folder and index.html... How to do that? thank you.

**Comment by MilosPaunovic** [comment, marked as answer]
Take a look at the `distDir` in the `build` property of `quasar.conf.js`.
[https://quasar.dev/quasar-cli/quasar-conf-js#property-build](https://quasar.dev/quasar-cli/quasar-conf-js#property-build)

**Reply by mecjos** [reply]
it says "Folder where Quasar CLI should generate the distributables. Relative path to project root directory. " in description... my laravel app is not in spa's root directory..

**Comment by MilosPaunovic** [comment]
> it says "Folder where Quasar CLI should generate the distributables. Relative path to project root directory. " in description... my laravel app is not in spa's root directory..

You are right. In that case, it would probably be the easiest for you to make use of `afterBuild` property and transfer newly created files to desired folder.

**Reply by mecjos** [reply]
yes, and I already ask how to do it :))

**Reply by metalsadman** [reply]
```js
...
distDir: '.../.../relativePath/toWhere/yourLaravelFolderis',
...
```
https://www.grainge.org/pages/authoring/relative_paths/relative_paths.htm

Then use WebpackCopyPlugin to copy other files ie. .htaccess/robots/index.php/etc.. in to your public folder. If you have other files to copy that is.

**Reply by mecjos** [reply]
another point.. I want to copy index.html file after build into laravel's resource folder.. I need copy command I think.. how can i do it?

**Reply by mecjos** [reply]
also distDir  prop delete target folder first and then create it again.. I need it to replace build files into target folder.
